api/src/main/resources/org/openmrs/module/ugandaemrreports/reports/METRICS.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@dbaluku suggestions for format after looking at a generated report 

`{
    "facilityid": "gLhqifDs7hU",    
    "emrversion": "3.0.0-SNAPSHOT",  
    "date-run": "July 1, 2019",
    "encounters": {
      "art": 4810,
      "hts": 88,
      "eid": 41
    },
    "patients": {
        "ever-enrolled": "patientsactiveincare":1540.0,
        "supressed":2448.0,
        "non-supressed":158.0
    },
    "poc-location-service": {
        "reception":0.0,
        "triage":0.0,
        "counselor":0.0,
        "clinician":0.0,
        "lab":0.0,
        "pharamcy":0.0
    }
  }
  `